### PR TITLE
Pass URL Parameters on Redirect

### PR DESCRIPTION
**Pull request by tausifmuzaffar**

### CODE REVIEW

File: view/rooms/new/app.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ function getPathVariables(adr){
   }
 
   const paramType = qdata.roomType || '';
-  const paramActive = qdata.active === 'true';
+  const paramActive = qdata.active !== 'false';
```
Comment (iamigo) [inline view/rooms/new/app.js]: so the intended behavior here is for any value other than the string "false" to be treated as true?

Comment (tausifmuzaffar) [reply view/rooms/new/app.js]: yeah, default everything to active unless specifically false

**Comment by kamilsmuga** [comment]
@tausifmuzaffar build failure

**Comment by tausifmuzaffar** [comment]
@kamilsmuga @iamigo fixed build

File: view/rooms/new/app.js (hunk 1 above)
Comment (iamigo) [reply view/rooms/new/app.js]: I wouldn't object to a comment on this line saying just that ;)

**Pull request merged**